src/printer.js

```diff
@@ -4261,9 +4261,30 @@ function printAstToDoc(ast, options) {
     );
   }
 
-  const doc = printGenerically(FastPath.from(ast));
+  let doc = printGenerically(FastPath.from(ast));
+  if (addAlignmentSize > 0) {
+    // Add a hardline to make the indents take effect
+    // It should be removed in index.js format()
+    doc = addAlignmentToDoc(concat([hardline, doc]), addAlignmentSize, options.tabWidth);
+  }
   docUtils.propagateBreaks(doc);
   return doc;
 }
 
-module.exports = { printAstToDoc };
+function addAlignmentToDoc(doc, size, tabWidth) {
```

Comment:
Yup, fixed in https://github.com/prettier/prettier/pull/1609/commits/149e18ca87fe8f7a81e2d4da4ef2774299e7bbfe